test/presentation/agenda/agenda_view_model_test.dart

```diff
@@ -55,6 +47,124 @@ void main() {
     date: DateTime(2022, 8, 30, 15),
   );
 
+  test('when this week is empty but not next week', () {
+    // Given
+    final demarches = [demarcheSamediProchain];
+    final rendezvous = [rendezvousLundiProchain];
+    final store = givenState() //
+        .loggedInPoleEmploiUser()
+        .agenda(demarches: demarches, rendezvous: rendezvous, dateDeDebut: lundi22)
+        .store();
+
+    // When
+    final viewModel = AgendaPageViewModel.create(store);
+
+    // Then
+    expect(
+      viewModel.events,
+      [
+        WeekSeparatorAgendaItem("Semaine en cours"),
+        EmptyMessageAgendaItem("Pas de démarche ni de rendez-vous"),
+        WeekSeparatorAgendaItem("Semaine prochaine"),
+        RendezvousAgendaItem(rendezvousLundiProchain.id, collapsed: true),
+        DemarcheAgendaItem(demarcheSamediProchain.id, collapsed: true),
+      ],
+    );
+  });
+  test('when this week has one event but not next week', () {
```

Comment:
`when this week has events` (au début on n'en avait mis qu'un, ce qui n'est plus le cas)